webpack/assets/javascripts/react_app/routes/Models/ModelsPage/ModelsPage.js

```diff
@@ -1,74 +1,52 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button } from '@patternfly/react-core';
-import { Link } from 'react-router-dom';
 
 import { translate as __ } from '../../../common/I18n';
-import PageLayout from '../../common/PageLayout/PageLayout';
+import TableIndexPage from '../../../components/PF4/TableIndexPage/TableIndexPage';
```

Comment:
if I am not mistaken, this layout is not only for tables, should it leave also under /common
everything is changing to PF4 so moving into this folder and later we would probably switch to common won't make sense, how about:

```suggestion
import IndexPageLayout from '../../common/PageLayout/IndexPageLayout';
```

Reply:
I think as we add search, create new and export button here this page should be only used for tables